src/components/OptionsList/BaseOptionsList.js

```diff
@@ -219,6 +224,8 @@ function BaseOptionsList({
                     ) : null}
                     <SectionList
                         ref={innerRef}
+                        nestedScrollEnabled
```

Comment:
1. `nestedScrollEnabled` is **true** by default on iOS
2. `nestedScrollEnabled` is **false** by default on Android, and it was added in **API 21** which is exactly the lowest we support right now.
3. `nestedScrollEnabled` should have no effect on browsers.

Leaving it to **true** as default here would put both iOS and Android on the same line, and it should have no negative effect in the application for any platform. I've looked over places where this component is used, and couldn't find any new bugs/different behaviors.